### PR TITLE
DOC-698 updating for 1.2

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR updates the Spectro Proxy packs documentation page. 
The Kubernetes Dashboard page has been moved to cluster management in preparation for a future refactor related to integrated packs. 

This pack has also been refactored to conform with the [Packs layout](https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1800994817/Packs+Layout).

CI related. This PR updated the Netlify deployment configuration with the correct Algolia environment variables. The Prow workflow was also updated to not use the incorrect `build_doc` function as it used the API key for production Algolia.

💻 [Preview](https://deploy-preview-1084--docs-spectrocloud.netlify.app/integrations/frp)

### CODE REVIEW

**Comment by spectro-prow** [comment]
@karl-cardenas-coding: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 3df818da250e59bf79f7ccfd4e66d8517601b351 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1084/librarium_build/1615857303053930496) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@karl-cardenas-coding: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 6befcfefc86403e43472ae016f6847b5dab22246 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1084/librarium_build/1615865608753123328) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@karl-cardenas-coding: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 0725af740c9dfadc67c690561e6d9054d50186a7 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1084/librarium_build/1618350980326232064) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@karl-cardenas-coding: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | d3be806b42948186c0cd75fb27cadb608cfab565 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1084/librarium_build/1618352993600540672) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@karl-cardenas-coding: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | ed6c436a486da6157a21b8e685e40f04df387ca6 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1084/librarium_build/1618380550521753600) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/06-integrations/00-frp.md
Comment (ritawatson) [inline content/docs/06-integrations/00-frp.md:19]:
```suggestion
Spectro Proxy is a pack that enables the use of a reverse proxy with a Kubernetes cluster. The reverse proxy allows you to connect to the API of a Palette-managed Kubernetes cluster in private networks or clusters configured with private API endpoints. The reverse proxy managed by Spectro Cloud is also known as the forward reverse proxy (FRP).
```

File: content/docs/06-integrations/00-frp.md
Comment (ritawatson) [inline content/docs/06-integrations/00-frp.md:21]:
```suggestion
The reverse proxy has a server component and a client component. The reverse proxy server is publicly available and managed by Spectro Cloud. The client runs inside your Palette-managed Kubernetes cluster and connects to the reverse proxy server. When you add the Spectro Proxy pack to a cluster profile, a couple of things happen:
```

File: content/docs/06-integrations/00-frp.md
Comment (ritawatson) [inline content/docs/06-integrations/00-frp.md:25]:
```suggestion
- The KubeConfig file is updated with the reverse proxy address instead of pointing directly to the cluster's API address. The following is an example of a kubeconfig file where the `server` attribute points to the reverse proxy.
```

File: content/docs/06-integrations/00-frp.md
Comment (ritawatson) [inline content/docs/06-integrations/00-frp.md:40]:
```suggestion
- Any requests to the Kubernetes API server, such as kubectl commands, will be routed to the reverse proxy. The reverse proxy forwards the request to the intended client, which is the cluster's API server. The cluster's API server authenticates the request and replies with the proper response.
```

File: content/docs/06-integrations/00-frp.md
Comment (ritawatson) [inline content/docs/06-integrations/00-frp.md:43]:
```suggestion
You can attach this pack to a [cluster profile](/cluster-profiles). The pack installs the Spectro Proxy client in the workload clusters and configures the cluster's API server to point to a managed proxy server.
```

File: content/docs/06-integrations/00-frp.md
Comment (ritawatson) [inline content/docs/06-integrations/00-frp.md:51]: The "Enable Kubernetes Dashboard" link doesn't work. Is it because **https://** is missing in the path?

File: content/docs/06-integrations/00-frp.md
Comment (ritawatson) [inline content/docs/06-integrations/00-frp.md:62]: Is it our style to format port numbers with an inline code element? This format wasn't flagged in the 'Ingress' PR - thinking about consistency.

File: content/docs/06-integrations/00-frp.md
Comment (ritawatson) [inline content/docs/06-integrations/00-frp.md:87]:
```suggestion
To use this pack, you have to add it to your cluster profile.  You can also add the Spectro Proxy pack when you create the cluster profile. Check out the [Create Cluster Profile](/cluster-profiles/task-define-profile) guide to learn more about cluster profile creation.
```

File: content/docs/06-integrations/00-frp.md
Comment (ritawatson) [inline content/docs/06-integrations/00-frp.md:94]:
```suggestion
- **Pure**: A brand new IaaS cluster that is deployed or will be deployed through Palette. An IaaS cluster is a Kubernetes cluster with a control plane that is not managed by a third party or cloud vendor but is completely managed by Palette. Google GKE and Tencent TKE fall in this category. 
```

File: content/docs/06-integrations/00-frp.md
Comment (ritawatson) [inline content/docs/06-integrations/00-frp.md:96]:
```suggestion
- **Non-pure**: A non-IaaS cluster whose control plane is managed by a third party or will be managed by a third party. Azure AKS and AWS EKS fall in this category, as both Palette and the cloud provider partially manage the clusters. Clusters imported into Palette are also included in this category.
```

File: content/docs/06-integrations/00-frp.md
Comment (ritawatson) [inline content/docs/06-integrations/00-frp.md:137]:
```suggestion
Set the parameter `k8sDashboardIntegration.enabled` to true if you intend to expose the Kubernetes dashboard. 
```

File: content/docs/06-integrations/00-frp.md
Comment (ritawatson) [inline content/docs/06-integrations/00-frp.md:343]: Broken link - missing **https://** ?

**Review by ritawatson — APPROVED** [review]
Just some minor wording suggestions to remove awkwardness around possession in a couple places and to shorten some sentences or make them more active voice.

A bit of space could be added between text above a couple examples - unless this is intended to save space in a large topic.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1084#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1084#pullrequestreview-1270292878" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-frp.md
Comment (cloudmaniac) [inline content/docs/06-integrations/00-frp.md:32]: Typo here with a space between https: and //

File: content/docs/06-integrations/00-frp.md
Comment (cloudmaniac) [inline content/docs/06-integrations/00-frp.md:94]: We don't already support GKE, no?

**Review by cloudmaniac — COMMENTED** [review]
That's excellent and will be very helpful to our customers!

Maybe we should add a note that the user has to re-download the `kubeconfig` file if he enables the proxy as a day 2 operations? E.g.: he deploys a cluster w/o proxy, download the kubeconfig file, enables the proxy.

File: content/docs/06-integrations/00-frp.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-frp.md:32]:
```suggestion
    server: https://cluster-11111111111111.proxy.stage.spectrocloud.com:443
```

File: content/docs/06-integrations/00-frp.md
Comment (karl-cardenas-coding) [reply content/docs/06-integrations/00-frp.md:43]: Nice catch on this one 😄

File: content/docs/06-integrations/00-frp.md
Comment (karl-cardenas-coding) [reply content/docs/06-integrations/00-frp.md:62]: Let's remove the inline, no need to have it.

Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-frp.md:62]:
```suggestion
- Outbound internet connectivity for port 443 is allowed so that you and your applications can connect with the Spectro Cloud reverse proxy.
```

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1084#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1084#pullrequestreview-1270292878" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **zulfilee**
You can assign the PR to them by writing `/assign @zulfilee` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**
- **[prow/OWNERS](https://github.com/spectrocloud/librarium/blob/master/prow/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-frp.md
Comment (karl-cardenas-coding) [reply content/docs/06-integrations/00-frp.md:426]: Fixed, nice catch

File: content/docs/06-integrations/00-frp.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-frp.md:407]:
```suggestion
### x509 Unknown Authority Error
```

**Pull request merged**
